sig-vmware/charter.md

```diff
@@ -0,0 +1,58 @@
+- Determining and documenting best practices for configuring Kubernetes when running on vSphere infrastructure.
+- Determining and documenting best practices for configuring VMware infrastructure when supporting Kubernetes.
+- Determining and documenting best practices for using common configuration management and orchestration tools for deploying and managing Kubernetes on VMware infrastructure.
+- Discussing bugs and feature requests recorded as Kubernetes or VMware cloud provider issues on GitHub. These issues should be tagged with ``sig/vmware``.
+
+A directory of specialized subject area GitHub teams for issues, PRs, and design proposals is defined in [SIG GitHub Teams](https://github.com/kubernetes/community/tree/master/sig-vmware#github-teams).
+
+#### Code, Binaries and Services
+
+- vSphere Cloud Provider
+- vRealize Automation Cluster API Provider
+
+Code locations and projects are defined in [SIG Subprojects](https://github.com/kubernetes/community/tree/master/sig-vmware#subprojects)
+
+#### Cross-cutting and Externally Facing Processes
+
+SIG VMware serves to bring together members of the VMware and Kubernetes community to maintain, support and provide guidance for running Kubernetes on VMware platforms.
+
+The VMware SIG will provide a forum for hosting related architectural planning and discussion. Associated activities related to development, testing, and documentation will be tracked and reported by the SIG.
+
+- The vSphere platform’s availability and resource management can create pools that extend across Kubernetes worker node, management node, and namespace boundaries. Unique features of the vSphere platform related to high availability, load balancing, and resource management (storage, networking, compute) can supplement existing capabilities of Kubernetes, container runtime, and OS functionality.
+  - Users can do some of this via configuration today, but opportunities exist to achieve more efficient, predictable, and dependable service through additional new CRDs, plug-ins, and enhancement proposals (KEPs) related to other Kubernetes components.
+  - Interaction is expected to cut across multiple aspects of Kubernetes in ways not firmly bounded by existing SIG dividing lines (e.g. node, scheduling, scalability).
+
+### Out of scope
+
+The VMware SIG is not for discussing bugs or feature requests outside the scope of Kubernetes. For example, the VMware SIG should not be used to discuss or resolve support requests related to VMware products. It should also not be used to discuss topics that other, more specialized SIGs own (to avoid overlap).
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [sig-governance]
+and opts-in to updates and modifications to [sig-governance].
+
+### Subproject Creation
+
+Associated subprojects are created following the Federation of Subprojects procedure described in [sig-governance]

```

Comment:
You need to choose which option you're taking.

Reply:
The option shoices are Option 1: by SIG Technical Leads or Option 2: by Federation of Subprojects.. so if the election of the "Federation of Subproject" isn't clear now could you suggest better wording?

Reply:
ack, I misread, I have been used to sigs saying "Option 1" or "Option 2", this is fine